alerter/src/main/java/com/tapadoo/alerter/Alert.java

```diff
@@ -343,7 +343,16 @@ public void setIcon(@DrawableRes final int iconId) {
         iconDrawable.setColorFilter(Color.WHITE, PorterDuff.Mode.SRC_ATOP);
         ivIcon.setImageDrawable(iconDrawable);
     }
+ /**
+     * Set the inline icon for the Alert
+     *
+     * @param iconDrawable Drawable  of the icon to use in the Alert
+     */
 
```

Comment:
Remove this blank space - this will fail the checkstyles and won't build. Run `./gradlew check` from the command line to verify.